server/model_server/visual_model_server.py

```diff
@@ -45,7 +45,7 @@
 
 # Path to frozen detection graph. This is the actual model that is used for the object detection.
 SSD_mobile_path = SERVER_ROOT + \
-                  '../../weights/ssd_mobilenet_v1_coco_2018_01_28/frozen_inference_graph.pb'
+                  '../../weights/ssd_mobilenet_v1_coco_2017_11_17/frozen_inference_graph.pb'
```

Comment:
It supposed to be 2018 version, Is the pre-trained weight tar file not updated?

Reply:
yes, it is the 2017 version

Reply:
I will update this